nodl_python/README.md

```diff
@@ -0,0 +1,2 @@
+# nodl_python: tools for parsing and analyzing nodl files
+![test nodl_python](https://github.com/ubuntu-robotics/nodl/workflows/test%20nodl_python/badge.svg)
```

Comment:
If we eventually have nodl_cpp in here as well, how will that change things? Will we need two badges? Or can we do something now that will make sure we only ever have one, even testing multiple packages?

Reply:
We can make one big `test` job, instead. If you notice, the workflow is specifically `test_python.yml`, I was planning on a separate action in a separate file for the cpp interface.

Reply:
We've agreed to follow the example of rosbag2 and have one bit test job instead.